onnxruntime/core/providers/rocm/math/einsum_utils/einsum_auxiliary_ops.cc

```diff
@@ -2,6 +2,7 @@
 // Licensed under the MIT License.
 
 #include "core/providers/shared_library/provider_api.h"
+#include "core/providers/rocm/tunable/gemm.h"
```

Comment:
Could you please order these two header files?

Reply:
No.